api/graphql/tests/test_spaces.py

```diff
@@ -219,3 +232,34 @@ def test_regular_user_cannot_update(self):
         assert_that(content.get("errors")).is_not_none()
         self.space.refresh_from_db()
         assert_that(self.space.name_fi).is_equal_to("Space1")
+
+
+class SpacesQueryTestCase(GrapheneTestCaseBase, snapshottest.TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.space = SpaceFactory(surface_area=40.5)
```

Comment:
Let's add some constant `name` for the space so that it will stay the same through separate test runs. 🙂

Reply:
that's fixed now